tests/js/spec/components/projectLink.spec.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import ProjectLink from 'app/components/projectLink';
+
+const path = 'http://some.url/';
+
+describe('ProjectLink', function() {
+  it('has environment in query', function() {
+    const wrapper = shallow(<ProjectLink to={path}>Go somewhere!</ProjectLink>, {
+      context: {
+        location: {
+          query: {
+            environment: 'staging',
```

Comment:
can we also add one where environment = ''?

Reply:
yup, sure